packages/special-pages/pages/onboarding/app/environment.js

```diff
@@ -0,0 +1,153 @@
+import { createContext, h } from 'preact'
```

Comment:
This file represents a shared `Environment` implementation, in future PRs it'll be moved and shared.